src/gui/optionsdlg.cpp

```diff
@@ -1182,6 +1206,9 @@ bool OptionsDialog::useAdditionDialog() const
 void OptionsDialog::enableApplyButton()
 {
     applyButton->setEnabled(true);
+
+    // Verify if the share action button must be enabled
+    m_ui->comboRatioLimitAct->setEnabled(m_ui->checkMaxRatio->isChecked() || m_ui->checkMaxSeedingMinutes->isChecked());
```

Comment:
>Because it won't reenable the Apply button if you modify the seeding limit, just the comboRatioLimitAct, and then you wouldn't be able to apply your changes.

But you have connected it to `enableApplyButton()` too, isn't it?